checks.d/elastic.py

```diff
@@ -504,6 +508,9 @@ def _process_pending_tasks_data(self, data, config):
             self._process_metric(node_data, metric, *desc, tags=config.tags)
 
     def _process_stats_data(self, nodes_url, data, stats_metrics, config):
+        # retreive the cluster name from the data, and append it to the
+        # master tag list.	
+        config.tags.append("cluster_name:{}".format(data['cluster_name']))
```

Comment:
Should we also update `config.service_check_tags` ? (or did you exclude them on purpose ?)